boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpuapp_common.dtsi

```diff
@@ -155,8 +182,13 @@
 
 		spi-max-frequency = <8000000>;
 	};
+
 	cs47l63: cs47l63@1 {
+		compatible = "cirrus,cs42l73";
 		reg = <1>;
+		irq-gpios = <&gpio0 19 GPIO_ACTIVE_LOW>;
+		reset-gpios = <&gpio0 18 GPIO_ACTIVE_LOW>;
+		gpio9-gpios = <&gpio0 20 GPIO_ACTIVE_LOW>;
```

Comment:
Suggest codec-to-nrf-gpios

Reply:
these are codec pins, it is named "GPIO9". How it's used is application choice.

Reply:
![image](https://user-images.githubusercontent.com/25011557/224320108-6190bdfa-3061-4de3-88fd-3eca646fa125.png)
![image](https://user-images.githubusercontent.com/25011557/224320185-f3e4917b-3f00-433f-af93-0e458e787109.png)

Reply:
ok, this is fine.